### PR TITLE
Allow survey questions to be edited and deleted

### PR DESCRIPTION
**Pull request by kxmbrian**
Option images in the question creation form can also be updated instead of having to delete and recreate a new option.

### CODE REVIEW

File: client/app/bundles/course/survey/containers/ShowQuestion.jsx [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+  deleteFailure: {
+    id: 'course.surveys.Question.deleteFailure',
+    defaultMessage: 'Failed to delete question.',
+  },
+});
+
+class ShowQuestion extends React.Component {
+  constructor(props) {
+    super(props);
+    this.updateQuestionHandler = this.updateQuestionHandler.bind(this);
```
Comment (allenwq) [inline client/app/bundles/course/survey/containers/ShowQuestion.jsx]: I find that a shorthand for defining bind method is:
```
updateQuestionHandler = (data) => {

}
```

in the above case, `updateQuestionHandler` will autobind to the current object.

**Comment by allenwq** [comment]
@kxmbrian Shall we use `.jsx` only for files that includes React template and `.js` for all other files ?  now everything is named `.jsx`.

File: client/app/bundles/course/survey/containers/ShowQuestion.jsx (hunk 1 above)
Comment (allenwq) [inline client/app/bundles/course/survey/containers/ShowQuestion.jsx]: @kxmbrian Are you going update this since #2005 is merged ?

**Comment by kxmbrian** [comment]
@allenwq thanks for the feedback. Updated!

**Comment by allenwq** [comment]
👍

**Pull request merged**